Source/Core/Common/Logging/ConsoleListener.h

```diff
@@ -14,5 +14,5 @@ class ConsoleListener : public Common::Log::LogListener
   void Log(Common::Log::LogLevel level, const char* text) override;
 
 private:
-  [[maybe_unused]] bool m_use_color = false;
+  bool m_use_color = false;
```

Comment:
This reintroduces a warning on android, see [this build log](https://dolphin.ci/#/builders/15/builds/6693/steps/2/logs/stdio) (it doesn't appear on the more recent build log because the file was unchanged between the two revisions of your pull request).  See #10388.

Reply:
Ah geez, warning tug-o-war.  So this member is for specific platforms?  Might it be better to wrap it in some preprocessor directives?

Reply:
Yeah, it's only for ConsoleListenerNix.  Based on this:

https://github.com/dolphin-emu/dolphin/blob/dcf8bd7733ae6393b879f4795b46104d67bba651/Source/Core/Common/CMakeLists.txt#L178-L197

checking if both `ANDROID` and `WIN32` are not defined would work.